package.json

```diff
@@ -2,6 +2,7 @@
   "name": "knp-react-app",
   "version": "0.1.0",
   "dependencies": {
+    "node-sass-tilde-importer": "^1.0.2",
```

Comment:
Can you move this dep to the `devDependencies` please ?
We need this to build the app, and so it's not part of the prod requirements ;)

Reply:
Dependency management is always disturbing for me. 
I understand that we should put `node-sass-tilde-importer` in *devDependencies*, but why not `node-sass-chokidar` ? 
We use it to compile sass files, so we use it only for building the app

Reply:
`node-sass-chokidar` is already in dev deps  (see https://github.com/KnpLabs/knp-react-app/blob/97ea5d679265e1e9aee8896932226ee41482c70a/package.json#L28 ) ;)

> We use it to compile sass files, so we use it only for building the app

exactly, this is used before shipping the bundled app to prod, and so is not necessary to run the app in prod (ie you do not import any module of this package when running the app in prod ;) ).

Reply:
Ok my bad :smile: